Gemfile

```diff
@@ -0,0 +1,5 @@
+source "https://rubygems.org"
+
+group :jekyll_plugins do
+  gem "jekyll-remote-theme", github: "benbalter/jekyll-remote-theme", branch: "master"
```

Comment:
Don't think you need the explicit reference to the gem, as I believe it's part of the default https://github.com/github/pages-gem/
I think you just need to set it in the `_config.yml` https://stackoverflow.com/questions/49116592/customize-jekyll-remote-theme-for-github-pages

Reply:
I think so too, but I'm trying to trigger the GitHub Action to build Jekyll and didn't know what should be in the Gemfile.

Reply:
Gotcha, might want to do the `github-pages` gem then, like https://github.com/canada-ca/ore-ero/blob/master/Gemfile